### ISSUE TITLE
Some Resource get URLS return HTML webpage instead of valid .jar plugin file

### ISSUE THREAD

**Issue by bjornpijnacker** [SpiGetOrg/Spiget#45]
As the title names, some get requests to `https://api.spiget.org/v2/resources/{plugin_id}/download` return the HTML download page for the plugin instead of the plugin's .jar file. Example IDs for which this occurs is 9089 (EssentialsX) and 274 (DynMap).

**Comment by InventivetalentDev** [comment]
both of those resources are hosted on external sites, hence the broken download. see #20 #27

**Comment by Th3Shadowbroker** [comment]
Hey there,
I were able to replicate the described behaviour. However both of the plugins you've named link to an external download page or at least their current version does. As these pages vary for each plugin, individual web-crawling is impossible. I've checked the request via postman and SpiGet responds with a [302 - Found](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/302). I think, that such a response should be handled on the client side as the API response matches [its documentation](https://spiget.org/documentation/#!/resources/get_resources_resource_download).

EDIT: Didn't saw the reply until GitHub reloaded when posting the comment 😄
